Makefile

```diff
@@ -94,6 +94,21 @@ fix-content:
 	curl -b cookie-jar -XPOST $(AULA_MK_CONTENT_URL)/api/manage-state/fix
 	rm -f cookie-jar
 
+testrenameclass:
```

Comment:
i get this in the logfile when running this:

```
2017-02-03 15:44:58.427813 UTC [ERROR] [QN6NaFSUP6b9Y66FCvHn9vuo] 500 Ill-formatted CSRF Token CsrfToken {fromCsrfToken = ""}
```

cookie-jar looks fine, from what i can tell.